XVim2/XVim/NSTextStorage+VimOperation.m

```diff
@@ -148,16 +149,20 @@ - (NSUInteger)xvim_lineNumberAtIndex:(NSUInteger)index
 
 #pragma mark Converting between Indexes and Line Numbers + Columns
 
-static NSUInteger xvim_sb_count_columns(xvim_string_buffer_t* sb, NSUInteger tabWidth)
+static NSUInteger xvim_sb_count_columns(xvim_string_buffer_t* sb, NSUInteger tabWidth, BOOL useTabToIndent)
 {
     NSUInteger col = 0;
 
     if (!xvim_sb_at_end(sb)) {
         do {
             if (xvim_sb_peek(sb) == '\t') {
-                col += tabWidth;
+                if (useTabToIndent) {
+                    col += 1;
+                } else {
+                    col += tabWidth;
```

Comment:
Actually I think we always need to increase col jut by one, as `\t` always count just as one character in Xcode. I left it like this for jut in case.  Maybe somebody can review and test and can be changed to += 1 for all cases.